packages/mobile/src/send/Send.tsx

```diff
@@ -215,13 +214,15 @@ class Send extends React.Component<Props, State> {
   }
 
   tryImportContacts = async () => {
-    const { numberVerified, allRecipients } = this.props
+    // CB TEMPORARY HOTFIX: Disabling phone number confirmation requirement
```

Comment:
Are we concerned that allowing non-verified users to send to phone numbers will lead to many out-of-quota errors? Maybe the ODIS quota params have changed since I checked but they were very strict for unverified numbers.

Reply:
They were at 2 before yesterday, but it was decided to increase the limit temporarily to 10, so only users that try to send to more than 10 numbers will run into this issue.